src/content/blog/my-attempt-at-linq-pagination.md

```
@@ -0,0 +1,60 @@
+	return recordSet;
+}
+```
+
+<p>What’s nice about the following code is that since LINQ is lazy loading, the SQL built doesn’t actually execute until we need it too.&#160; The days of returning full datasets are done (yes, we’re still doing that on some projects).&#160; </p>
+
+<p>I went the next step to see if I could build an extension method of my own that did all of the above for me automatically.&#160; Here was the result I came up with:</p>
+
+```csharp
+public static class ExtensionMethods
+{
+	public static IQueryable<t> Paginate<t>(this IQueryable<t> content, int pageNumber, int pageSize)
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)